Projects/App/Sources/UI/Place/PlaceList/View/PlaceListViewController.swift

```diff
@@ -42,10 +120,51 @@ extension PlaceListViewController {
         tableView.separatorStyle = .none
     }
     
+    private func removeBackgroundAndDivider() {
+        let removeBackgroundDivider = UIImage()
+        segmentedControl.setBackgroundImage(removeBackgroundDivider, for: .normal, barMetrics: .default)
+        segmentedControl.setBackgroundImage(removeBackgroundDivider, for: .selected, barMetrics: .default)
+        segmentedControl.setDividerImage(removeBackgroundDivider, forLeftSegmentState: .selected, rightSegmentState: .normal, barMetrics: .default)
+    }
+    
     private func createLayout() {
-        view.addSubview(tableView)
+        view.addSubviews([headerView, segmentedControl, segmentIndicator, questionMarkImage, tableView])
+        headerView.addSubviews([segmentedControl, segmentIndicator, questionMarkImage, sideButtonImage])
+        
+        headerView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top)
+            make.horizontalEdges.equalToSuperview()
+            make.height.equalTo(85)
+        }
+        
+        segmentedControl.snp.makeConstraints { make in
+            make.verticalEdges.equalToSuperview().inset(27)
+            make.left.equalToSuperview().inset(20)
+            make.width.equalTo(172)
+        }
```

Comment:
p5;

얘도 높이로? 근데 얘는 이미지같이 짜부되는 애는 아니라 뭐가 좋을지 잘 모르겠네요...

```suggestion
        segmentedControl.snp.makeConstraints { make in
            make.centerY.equalToSuperview()
            make.left.equalToSuperview().inset(20)
            make.width.equalTo(172)
            make.height.equalTo(32)
        }
```

Reply:
7281c930c08eb29f3a068d666e5166dd3e56c9bc